src/CoreBundle/private/library/classes/TFullGroupTable/TFullGroupTable.class.php

```diff
@@ -68,13 +68,17 @@ class TFullGroupTable extends TGroupTable
     public $pageingLocation = 'top_and_bottom';
 
     /**
+     * @deprecated since 6.3.0 - We don`t need to configure an icon URL anymore. We use an icon font.
```

Comment:
Could you add these deprecated fields to the UPGRADE file?

Reply:
done